website/docs/reference/project-configs/config-version.md

```diff
@@ -2,6 +2,12 @@
 datatype: integer
 ---
 
+<VersionBlock firstVersion="1.5">
+
+Starting in dbt v1.5, the `config-version:` tag is optional.
```

Comment:
```suggestion
Starting in dbt v1.5, the `config-version:` tag is now optional and is no longer required. For projects on v1.4 or lower, select the documentation version in the header and refer to the listed specifications.

Reply:
Is there another way we can word this? Nothing on the page will change if they switch versions other than the statement marking this as optional.

Reply:
oh i see, i think this is where i was confused - i assume the page contents changed and they would only see the '1.5 is optional' bit if they were on v1.5. and not see the other definitions, etc. it's a nonblocker for me - the main curiosity was the question about what it means for cloud and whether we should link to something else (migration guide) in the statement.